libs/partners/ollama/langchain_ollama/chat_models.py

```diff
@@ -514,15 +515,26 @@ def _create_chat_stream(
 
         params["options"]["stop"] = stop
         if "tools" in kwargs:
-            yield self._client.chat(
-                model=params["model"],
-                messages=ollama_messages,
-                stream=False,
-                options=Options(**params["options"]),
-                keep_alive=params["keep_alive"],
-                format=params["format"],
-                tools=kwargs["tools"],
-            )
+            if len(kwargs["tools"]) == 0:
```

Comment:
(nit) Wondering if there are any options for simplifying this:
- change `if "tools" in kwargs` to `if kwargs.get("tools")`, which is Falsey for empty list; or
- There's a method [_should_stream](https://github.com/langchain-ai/langchain/blob/00e7b2dada9085aad12ec47e7bd84089bad1d2bc/libs/core/langchain_core/language_models/chat_models.py#L362C21-L362C35) which is called on both `.stream` and `.astream` that controls whether we delegate to `.invoke`. Could potentially override this on ChatOllama. See example here: https://github.com/langchain-ai/langchain/blob/00e7b2dada9085aad12ec47e7bd84089bad1d2bc/libs/partners/openai/langchain_openai/chat_models/base.py#L982-L1002